C_routing.md

```diff
@@ -301,58 +300,69 @@ end
 
 Note that Phoenix will assume that the path we set ought to begin with a slash, so `scope "/admin" do` and `scope "admin" do` will both produce the same results.
 
-Note also, that the way this scope is currently defined, we need to fully qualify our controller name, `HelloPhoenix.Admin.ReviewController`. We'll fix that in a minute.
+Note also, that the way this scope is currently defined, we need to fully qualify our controller name, `Admin.ReviewController`. We'll fix that in a minute.
+
+Running `$ mix phoenix.routes` again, in addition to the previous set of routes we get the following:
 
 ```elixir
+. . .
 review_path  GET     /admin/reviews           HelloPhoenix.Admin.ReviewController.index/2
 review_path  GET     /admin/reviews/:id/edit  HelloPhoenix.Admin.ReviewController.edit/2
 review_path  GET     /admin/reviews/new       HelloPhoenix.Admin.ReviewController.new/2
 review_path  GET     /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.show/2
 review_path  POST    /admin/reviews           HelloPhoenix.Admin.ReviewController.create/2
-review_path  PUT     /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.update/2
-             PATCH   /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.update/2
+review_path  PATCH   /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.update/2
+             PUT     /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.update/2
 review_path  DELETE  /admin/reviews/:id       HelloPhoenix.Admin.ReviewController.delete/2
 ```
 
-This looks good, but there is a problem here. Remember that we wanted both user facing reviews routes as well as the admin ones. When we define both of those routes in our router, like this,
+This looks good, but there is a problem here. Remember that we wanted both user facing reviews routes `/reviews` as well as the admin `/admin/reviews` ones. If we now include the user facing reviews in our router like this:
```

Comment:
I would switch the order of `/admin/reviews` and 'ones' like this: 

. . . as well as the admin ones `/admin/reviews`.